README.md

```diff
@@ -94,35 +96,17 @@ For a comprehensive reference of the language, check out the [Wing Language Spec
 
 ## FAQs ❓
 
-### What makes Wing a good fit for cloud development? 🌟
-
-Wing was built from the ground up to make it an ideal choice for building applications on AWS or other cloud providers.
-Some of these features include:
-
-* [Cloud services](https://docs.winglang.io/concepts/resources) as first-class citizens.
-* Higher level of cloud abstraction with a [standard library](https://docs.winglang.io/reference/wingsdk-spec).
-* [Distributed computing primitives](https://docs.winglang.io/concepts/inflights).
-* [Compiler plugins](https://docs.winglang.io/reference/compiler-plugins) for customization.
-* [JavaScript interoperability](https://docs.winglang.io/reference/spec#5-interoperability).
-* Automatic generation of IAM policies.
-* [Native JSON](https://docs.winglang.io/reference/spec#114-json-type) and schema validation support.
-
-For a more in-depth look at Wing's features and benefits, check out our [documentation](https://docs.winglang.io/).
+Have a question about Wing? head over to our [FAQ](https://docs.winglang.io/faq) to find the answer.
+Three main questions you can find there are:
```

Comment:
(optional) Also if you have time I think adding a FAQ on the language's goals could be really cool. Maybe we could write something based on some of the original pain points that we discussed long ago:

```
Too low level - Low level programming model which requires tedious effort to define many small resources.

Poor local emulation - Local development doesn't emulate deployed systems well enough to make for reliable local testing

Slow feedback loop - Because local development is broken, you must deploy to running environment which causes long feedback loops.

Poor testing experience - Most cloud architectures are now defined in IaC which is inherently just static code, which makes testing fragile and unreliable. Testing deployed systems becomes difficult because of the slow feedback loop.

Physically separated but logically co-dependent concerns - DevOps owns non-functional concerns while developers own functional concerns, but one can't be built without the other so there's always a chicken & egg problem that slows down the development process.

Leaky models - Leaky functional concerns which requires developer to consider implementation before design.

Lack of refactoring tools - Cloud Architectures are typically hard to refactor into reusable components because of a lack of good tooling, testable code, and systems built from implementation-first designs that often fail to consider re-usability.
```

Reply:
> (optional) Also if you have time I think adding a FAQ on the language's goals could be really cool. Maybe we could write something based on some of the original pain points that we discussed long ago:

Good idea, but for the next PR :)

Reply:
> Makes sense, but I would also add the "why a language" question to that list. You would put these two sections outside of the FAQ section?

I guess it depends how much space it takes. If you want to write several long paragraphs about why a language, then I think FAQ is better, but if there is a summary we can include in the README that is good too 👍